Configuration/templates/designToConfigurator.jinja

```diff
@@ -98,85 +96,68 @@ using namespace CalculatedVariables;
+          const auto& xmlObj(config.{{innerClass}}()[xmlIndex]);
+          LOG(Log::DBG)<<__FUNCTION__<<" Configuring class type [id:"<<xmlType<<", nm:{{innerClass}}] ordering index ["<<xmlIndex<<"] parent class type []";
+          {% if not designInspector.class_has_device_logic(innerClass) %}
+            // class [{{innerClass}}] has no device logic: configure only address space object
+            configure{{innerClass}} (xmlObj, nm, {{parentNodeId}});
+          {% else %}        
+            // class [{{innerClass}}] has device logic: configure device object
+            {% if designInspector.class_has_legit_device_parent(innerClass) %}
+              Device::Parent_D{{innerClass}}* pInnerItemParent = {{parentDevice}};
+            {% else %}
+              Device::Parent_D{{innerClass}}* pInnerItemParent = nullptr;
+            {% endif %}
+            auto dInnerObj = configure{{innerClass}}(xmlObj, nm, {{parentNodeId}}, pInnerItemParent);
+  
+            // register class {{innerClass}}] with parent (or register orphan)
+            {% if ((designInspector.class_has_device_logic(parentClassName)) or ('Root' == parentClassName)) %}
+              {{parentDevice}}->add(dInnerObj);
+            {% else %}
+              Device::D{{innerClass}}::registerOrphanedObject(dInnerObj);
+           {% endif %}
+          {% endif %}        
         }
-        {% if designInspector.class_has_device_logic(innerClass) %}
-          // class [{{innerClass}}] has device logic: Configure device object and register with parent
-          {% if designInspector.class_has_legit_device_parent(innerClass) %}
-            Device::Parent_D{{innerClass}}* pInnerItemParent = {{parentDevice}};
-          {% else %}
-            Device::Parent_D{{innerClass}}* pInnerItemParent = nullptr;
-          {% endif %}
-          Device::D{{innerClass}}* dInnerItem = configure{{innerClass}} (
-            *innerObjectConfig, nm, {{parentNodeId}}, pInnerItemParent);
-
-          {% if ((designInspector.class_has_device_logic(parentClassName)) or ('Root' == parentClassName)) %}
-            {{parentDevice}}->add(dInnerItem);
-          {% else %}
-            Device::D{{innerClass}}::registerOrphanedObject(dInnerItem);
-         {% endif %}
-        {% else %}
-          // class [{{innerClass}}] has no device logic. Configure only address space object
-          configure{{innerClass}} (*innerObjectConfig, nm, {{parentNodeId}});
-      {% endif %}
-      continue;
+        break;
+      {% endfor %}
+      case Configuration::{{xsdParentType}}::CalculatedVariable_id:
+      {
+        const auto& xmlObj(config.CalculatedVariable()[xmlIndex]);
+        LOG(Log::DBG)<<__FUNCTION__<<" Configuring type [id:"<<xmlType<<", nm:CalculatedVariable] ordering index ["<<xmlIndex<<"]";
+        Engine::instantiateCalculatedVariable(nm, {{parentNodeId}}, xmlObj);
```

Comment:
Could we NOT cut-off the namespace here? Prependign with CalculatedVariables:: will, IMO, but more readable.

Reply:
Not sure I understand this point. Which line do you refer to ?

Reply:
line 168.

Reply:
Ah I see (_using namespace_ was a straight copy from XSLT). Done
https://github.com/quasar-team/quasar/commit/cf549cbae2cbf1208186d8edaa979b265b6ddf94